### ISSUE TITLE
group-by agg sum crashes 

### ISSUE THREAD

**Issue by yingzhu146** [nushell/nushell#5764]
### Related problem

 Testing out the new prefix-less df commands, this simple aggregation crashes

```
ls  | to-df  | group-by type  | agg ( col size  | sum ) 

```

### Describe the solution you'd like

Not crash :)

### Additional context and details

Full strace:

<details>
  <summary>

</summary>

```
ls  | to-df  | group-by type  | agg ( col size  | sum )                                                                                                                                  06/12/2022 10:58:38 PM
thread '<unnamed>' panicked at 'not implemented', /Users/yingzhu/.cargo/registry/src/github.com-1ecc6299db9ec823/polars-core-0.21.1/src/series/series_trait.rs:76:13
stack backtrace:
   0:        0x105d7f750 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h05a395e187ce2aa0
   1:        0x104eda0dc - core::fmt::write::h05644aebdb29027a
   2:        0x105d787b8 - std::io::Write::write_fmt::h21aca3a0318ef6d9
   3:        0x105d81268 - std::panicking::default_hook::{{closure}}::h1fde348d0e5820ac
   4:        0x105d81014 - std::panicking::default_hook::h3054919319bc1fc2
   5:        0x105d81760 - std::panicking::rust_panic_with_hook::h432fb8a3a113cf66
   6:        0x105d815b4 - std::panicking::begin_panic_handler::{{closure}}::h7f35659f3b4db1de
   7:        0x105d7fb98 - std::sys_common::backtrace::__rust_end_short_backtrace::hc20c188030fdb5cf
   8:        0x105d81374 - _rust_begin_unwind
   9:        0x105f89290 - core::panicking::panic_fmt::hb7a7cffdeb3d6b1f
  10:        0x105f891b0 - core::panicking::panic::hf9f7eb4e327a15bb
  11:        0x10508d3c4 - polars_core::series::series_trait::private::PrivateSeries::agg_sum::h705d0f20cd127259
  12:        0x105b70f74 - <polars_lazy::physical_plan::expressions::aggregation::AggregationExpr as polars_lazy::physical_plan::expressions::PhysicalAggregation>::aggregate::h9ad092a877553165
  13:        0x105b73604 - polars_lazy::physical_plan::expressions::utils::as_aggregated::h1377940d5e35e2fe
  14:        0x105b01460 - core::ops::function::impls::<impl core::ops::function::FnMut<A> for &F>::call_mut::hfc9464a4fe2a846b
  15:        0x105ac4b74 - alloc::vec::Vec<T,A>::extend_desugared::he233abffedef5135
  16:        0x105aef0e4 - <rayon::iter::extend::ListVecFolder<T> as rayon::iter::plumbing::Folder<T>>::consume_iter::hccad41d409de107d
  17:        0x105b388a0 - <rayon::iter::while_some::WhileSomeFolder<C> as rayon::iter::plumbing::Folder<core::option::Option<T>>>::consume_iter::h0c6e7201226fb9d8
  18:        0x105b0fb04 - rayon::iter::plumbing::bridge_producer_consumer::helper::h78ed2dd4b6b59234
  19:        0x105b0de60 - rayon::iter::plumbing::bridge_producer_consumer::h1950b494571c5c8b
  20:        0x105b437c4 - <rayon::iter::while_some::WhileSome<I> as rayon::iter::ParallelIterator>::drive_unindexed::h508def4f54587854
  21:        0x105b911c0 - rayon::iter::from_par_iter::collect_extended::ha0ff799c2cd8c97e
  22:        0x105b569f0 - rayon::result::<impl rayon::iter::FromParallelIterator<core::result::Result<T,E>> for core::result::Result<C,E>>::from_par_iter::hb41e007794b459c2
  23:        0x105afd274 - std::panicking::try::h498ac42e11e457c9
  24:        0x105b22e90 - <rayon_core::job::StackJob<L,F,R> as rayon_core::job::Job>::execute::hfe4948145ec1a6d4
  25:        0x105fc0048 - rayon_core::registry::WorkerThread::wait_until_cold::h94ba96c0b67426b4
  26:        0x105bda878 - rayon_core::registry::ThreadBuilder::run::hb4846d2b0579fa3f
  27:        0x105bdc63c - std::sys_common::backtrace::__rust_begin_short_backtrace::ha432b117b43f88f9
  28:        0x105bdc6bc - std::panicking::try::hd3c982c88e3a6dc6
  29:        0x105bd8bfc - core::ops::function::FnOnce::call_once{{vtable.shim}}::h538a1872baaef733
  30:        0x105d86358 - std::sys::unix::thread::Thread::new::thread_start::h9d5932076be6950f
  31:        0x1954b226c - __pthread_deallocate
```

</details>

|version|branch|commit_hash|build_os|build_target|rust_version|rust_channel|cargo_version|pkg_version|build_time|build_rust_channel|features|installed_plugins|
|-|-|-|-|-|-|-|-|-|-|-|-|-|
|0.63.1||11d7d8ea1e83cc258141a9d97145de851238bdb6|macos-aarch64|aarch64-apple-darwin|rustc 1.62.0-nightly (055bf4ccd 2022-04-25)|nightly-aarch64-apple-darwin|cargo 1.62.0-nightly (edffc4ada 2022-04-19)|0.63.1|2022-06-12 22:26:34 -04:00|release|database, dataframe, default, trash, which, zip||

**Comment by fdncred** [comment]
This example works now. closing.
```
ls | to-df | group-by type | agg (col size | sum) 
╭──────┬────────╮
│ type │  size  │
├──────┼────────┤
│ file │ 167839 │
│ dir  │  24576 │
├──────┼────────┤
│ type │  size  │
╰──────┴────────╯
```

**Comment by jntrnr** [comment]
@yingzhu146 - does your directory contain things that aren't files or directories (for example, symlinks or other types of entries?)

**Comment by yingzhu146** [comment]
Yes likely - I can't unfortunately test this fix since I'm running into #5330 again, however, this time none of these env vars exist.
